doc/Advanced.md

```diff
@@ -502,6 +504,41 @@ Feel free to consult the existing usage examples among other modules in the QC r
 
 Once a custom class is implemented, one should let QCG know how to display it correctly, which is explained in the subsection [Display a non-standard ROOT object in QCG](#display-a-non-standard-root-object-in-qcg).
 
+## Critical and non-critical tasks
+
+Some DPL devices can be marked as non-critical. It means that if they die the system will continue running. There is 
+obviously an impact as all the downstream devices won't get data. 
+
+### QC tasks
+
+In QC, one can mark a task as critical or non-critical:
+```json
+    "tasks": {
+      "QcTask": {
+        "active": "true",
+        "critical": "false",     "": "if false the task is allowed to die without stopping the workflow, default: true",
+```
+By default, they are critical. 
+
+### QC mergers
+
+Mergers are critical or not based on the criticality of the task they are merging data for. 
+
+### QC checkers
+
+Checkers are non-critical. 
```

Comment:
Do you mean Check Runners or Checks?

Reply:
Anyway, they also seem to be critical, since they can survive a missing input, but if they die, it should stop the workflow, no?